kfdefs/overlays/moc/smaug/opf-jupyterhub-stage/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: opf-jupyterhub-stage
+
+resources:
+  - ../../../../base/jupyterhub
+
+patchesJson6902:
+  - patch: |
+      - op: add
+        path: /spec/applications/2/kustomizeConfig/overlays/-
+        value: storage-class
+      - op: add
+        path: /spec/applications/2/kustomizeConfig/parameters/-
+        value:
+          name: storage_class
+          value: oc-nfs-csi
```

Comment:
this needs to be updated to `moc-nfs-csi`

Reply:
Can we just drop the storageclass and let it use the default?

Reply:
we could do that :+1: